bin/main.rb

```diff
@@ -1,34 +1,74 @@
 #!/usr/bin/env ruby
 
-puts "-------------Welcome to Tic Tak Toe-----------------"
-print "Enter Player 1 name: "
+require_relative '../lib/game_logic.rb'
+require_relative '../lib/player.rb'
+require_relative '../lib/board.rb'
 
-player_1 = gets.chomp.capitalize
+puts '-------------Welcome to Tic Tak Toe-----------------'
+print 'Enter Player 1 name: '
 
-puts "The Player 1 is #{player_1}"
+player_one = gets.chomp.capitalize
 
-print "Enter Player 2 name: "
+puts "The Player 1 is #{player_one}"
 
-player_2 = gets.chomp.capitalize
+print 'Enter Player 2 name: '
 
-puts "The Player 2 is #{player_2}"
+player_two = gets.chomp.capitalize
 
-puts "Player 1 Please Select Your Symbol X or O"
+puts "The Player 2 is #{player_two}"
 
-player_1_sym = gets.chomp.upcase
+puts 'Player 1 Please Select Your Symbol X or O'
 
-until player_1_sym == "X" || player_1_sym == "O"
-  puts "Please choose the correct symbol X or O"
-  player_1_sym = gets.chomp.upcase
-end
+player_one_sym = gets.chomp.upcase
 
-if player_1_sym == "X"
-  player_2_sym = "O" 
-else
-  player_2_sym = "X"
+while (player_one_sym != 'X') && (player_one_sym != 'O')
+  puts 'Please choose the correct symbol X or O'
+  player_one_sym = gets.chomp.upcase
 end
 
-puts "#{player_1} is #{player_1_sym} and #{player_2} is #{player_2_sym}"
+player_two_sym = if player_one_sym == 'X'
+                   'O'
+                 else
+                   'X'
+                 end
```

Comment:
To keep your code clean, please move this to a method on the player Class and call it here.